indexing-service/src/test/java/org/apache/druid/indexing/common/task/HadoopIndexTaskTest.java

```diff
@@ -72,8 +72,8 @@ public void testCorrectInputSourceTypes()
     Assert.assertEquals(
         Collections.singleton(
             new ResourceAction(new Resource(
-                ResourceType.EXTERNAL,
-                "hadoop"
+                "hadoop",
```

Comment:
Perhaps this could be a static variable in `HadoopIndexTask` that can be reused here? It'd also clarify the difference between task type and input source type:
```
  public static final String INPUT_SOURCE_TYPE = "hadoop";
```

Reply:
good suggestion, fixed